tornjak-frontend/src/tables/table-body.tsx

```diff
@@ -48,7 +55,7 @@ class Body extends React.Component<BodyProp, BodyState> {
                                     cell.value)}
                             </TableCell>
                         ))}
-                        {this.props.entityType === "Agent" &&
+                        {this.props.entityType === "Agent" && this.TornjakHelper.checkRolesAdminUser(this.props.globalUserRoles) &&
```

Comment:
I am slightly confused why we need to checkRolesAdminUser here - both user types can view? Or is this for something else?

Reply:
This is for Admin to set Workload Attestor Plugin. Viewer can only view.